api/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+ENV SERVER_HOST=0.0.0.0
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
+COPY . /app
```

Comment:
The python source is already in `/api`; do you mean to suggest the python source as going in `/api/app`?